### PR TITLE
Reduces curielogger's verbosity

### PR DESCRIPTION
**Pull request by xavier-rbz**
Do not log every single request to stdout.

### CODE REVIEW

File: curiefense/curielogger/curielogger/main.go [hunk 1]
```diff
@@ -733,7 +733,6 @@ func (s grpcServerParams) StreamAccessLogs(x als.AccessLogService_StreamAccessLo
 			}
 
 			for _, l := range s.loggers {
-				log.Print(l)
```
Comment (flaper87) [inline curiefense/curielogger/curielogger/main.go:736]: whooops!

File: curiefense/images/confserver/bootstrap/confdb-initial-data/master/config/json/waf-signatures.json [hunk 2]
```diff
@@ -213,7 +213,7 @@
...
...hback|dbms_hs_passthrough|dbms_iot|dbms_job|dbms_ldap|dbms_libcache|dbms_lob|dbms_lock|dbms_logmnr|dbms_logmnr_cdc_publish|dbms_logmnr_cdc_subscribe|dbms_logmnr_d|dbms_metadata|dbms_mview|dbms_obfuscation_toolkit|dbms_odci|dbms_offline_og|dbms_offline_snapshot|dbms_olap|dbms_oracle_trace_agent|dbms_oracle_trace_user|dbms_outln|dbms_outln_edit|dbms_output|dbms_pclxutil|dbms_pipe|dbms_profiler|dbms_random|dbms_rectifier_diff|dbms_redefinition|dbms_refresh|dbms_repair|dbms_repcat|dbms_repcat_admin|dbms_repcat_instatiate|dbms_repcat_rgt|dbms_reputil|dbms_resource_manager|dbms_resource_manager_privs|dbms_resumable|dbms_rls|dbms_rowid|dbms_session|dbms_shared_pool|dbms_snapshot|dbms_space|dbms_space_admin|dbms_sql|dbms_stats|dbms_trace|dbms_transaction|dbms_transform|dbms_tts|dbms_types|dbms_url|dbms_utility|dbms_wm|dbms_xmlgen|dmbs_xmlquery|dbms_xmlsave|debug_extproc|outln_pkg|sdo_cs|sdo_geom|sdo_lrs|sdo_migrate|sdo_tune|utl_coll|utl_encode|utl_file|utl_http|utl_inaddr|utl_pg|utl_raw|utl_ref|utl_smtp|utl_tcp|utl_url|anydata|anydataset|anytype|all_arguments|user_arguments|dba_objects|all_objects|user_objects|sys_objects|dba_source|all_source|user_source|myappadmin\\.adduser)[\\s\\.\\(]",
```
Comment (flaper87) [inline curiefense/images/confserver/bootstrap/confdb-initial-data/master/config/json/waf-signatures.json:216]: Is this change correct? Can't really tell what the change is and the tests are failing.

**Review by flaper87 — COMMENTED** [review]
comment on the signatures change, otherwise LGTM

File: curiefense/images/confserver/bootstrap/confdb-initial-data/master/config/json/waf-signatures.json (hunk 2 above)
Comment (xavier-rbz) [reply curiefense/images/confserver/bootstrap/confdb-initial-data/master/config/json/waf-signatures.json:216]: it is actually *not* correct, I'll remove it. Nice catch.

**Review by flaper87 — APPROVED** [review]
🍝

**Pull request merged**